.github/workflows/publish-release.yml

```diff
@@ -180,3 +180,42 @@ jobs:
           component: ${{ matrix.component }}
           projects: ${{ matrix.projects }}
           sentry_token: ${{ secrets.SENTRY_AUTH_TOKEN }}
+
+  upload-deb-packages:
+    runs-on: ubuntu-24.04
+    if: >-
+      ${{
+        startsWith(inputs.release_name || github.event.release.name, 'gateway') ||
```

Comment:
We probably want the headless-client in here too no?

Reply:
So far, I've not built a package for it but we can add that later. Trying to get all the moving bits into place.